src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,174 @@
+package duke;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Duke {
+    private static final ArrayList<Task> TASKS = new ArrayList<>();
+
+    private static void handleList() {
```

Comment:
I noticed that you have not done the More-OOP level. Maybe you can try to extract out some code as Ui, Parser, Tasklist, Storage classes?

Reply:
Thank you, I will refactor the code soon.

Reply:
Once A-More-OOP is done, if you have the time, maybe you could use java docs to comment your code and what it does.